src/test/java/space/obminyashka/items_exchange/controller/AdvertisementControllerTest.java

```diff
@@ -102,4 +110,63 @@ void findPaginatedAsThumbnails_shouldReturnProperQuantityOfAdvertisementsThumbna
                 .andExpect(status().isOk())
                .andExpect(jsonPath("$.length()").value(advertisementRepository.findAll().size()));
     }
+
+    @Test
+    @WithMockUser(username = "admin")
+    void updateAdvertisement_shouldReturn400WhenNotValidAdvertisementFields() throws Exception {
+        AdvertisementDto existDtoForUpdate = AdvertisementDtoCreatingUtil
+                .createExistAdvertisementDtoForUpdateWithNotValidFields();
+
+        final var validationMessageSize =
+                createValidationMessage("size", existDtoForUpdate.getSize(), "1", "50");
+        final var validationMessageTopic =
+                createValidationMessage("topic", existDtoForUpdate.getTopic(), "3", "70");
+        final var validationMessageDescription =
+                createValidationMessage("description", existDtoForUpdate.getDescription(), "255");
+        final var validationMessageWhishes =
+                createValidationMessage("wishesToExchange", existDtoForUpdate.getWishesToExchange(), "210");
+        final var validationMessageCity =
+                createValidationMessage("location.city", existDtoForUpdate.getLocation().getCity(), "2", "100");
+        final var validationMessageArea =
+                createValidationMessage("location.area", existDtoForUpdate.getLocation().getArea(), "2", "100");
+        final var validationMessageDistrict =
+                createValidationMessage("location.district", existDtoForUpdate.getLocation().getDistrict(), "2", "100");
```

Comment:
Could you please replace all these variables into AdvertisementDtoCreatingUtil class

Reply:
ok, i`ll replace all it